README.md

```diff
@@ -1,4 +1,35 @@
 # `@ubiquity/onboard.ubq.fi`
 
-Generates config for organizations
+Generates the configuration for organizations, by creating a default configuration and creating a repository under the 
+given Organization.
 
+## Requirements
+
+Copy the `env.example` to `.env` and fill the required variables.
+
+## Run
+
+```shell
+yarn start
+```
+Should make the front-end page available at [http://localhost:8080](http://localhost:8080).
+
+### Required fields
+- `CHAIN_ID`: the id of the network you want to use for the transactions
+- `WALLET_PRIVATE_KEY`: the [64 digits](https://www.browserling.com/tools/random-hex) crypto wallet key that will be used for transactions
```

Comment:
What's up with the url

Reply:
Just offering a website to generate a random hex string for testing, can remove it if you want.